### PR TITLE
Bump reactor-netty to 1.0.0

### PR DESCRIPTION
**Pull request by Sm0keySa1m0n**

### CODE REVIEW

File: transport-parent/transport-netty/pom.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
     <dependency>
       <groupId>io.projectreactor.netty</groupId>
       <artifactId>reactor-netty</artifactId>
+      <version>1.0.0</version>
```
Comment (segabriel) [inline transport-parent/transport-netty/pom.xml:24]: Please revert it. Change the reactor-bom version in the main pom.xml:42 instead.

```
<reactor.version>2020.0.4</reactor.version>
```

File: transport-parent/transport-netty/src/main/java/io/scalecube/transport/netty/tcp/TcpReceiver.java [hunk 2]
```diff
@@ -42,9 +41,7 @@ private TcpServer newTcpServer(ReceiverContext context) {
         .option(ChannelOption.TCP_NODELAY, true)
         .option(ChannelOption.SO_KEEPALIVE, true)
         .option(ChannelOption.SO_REUSEADDR, true)
-        .bootstrap(
-            b ->
-                BootstrapHandlers.updateConfiguration(
-                    b, "inbound", new TcpChannelInitializer(config.maxFrameLength())));
+        .doOnChannelInit((connectionObserver, channel, remoteAddress) -> 
+            new TcpChannelInitializer(config.maxFrameLength()).accept(connectionObserver, channel));
```
Comment (segabriel) [inline transport-parent/transport-netty/src/main/java/io/scalecube/transport/netty/tcp/TcpReceiver.java:45]: Seems it's not enough (test are not passed), so need to reverse ordering of adding handlers in `TcpChannelInitializer`:

```java
    pipeline.addFirst(new ExceptionHandler());
    pipeline.addFirst(
        new LengthFieldBasedFrameDecoder(
            maxFrameLength, 0, LENGTH_FIELD_LENGTH, 0, LENGTH_FIELD_LENGTH));
    pipeline.addFirst(new LengthFieldPrepender(LENGTH_FIELD_LENGTH));
```

**Review by segabriel — CHANGES_REQUESTED** [review]
Also for some tests (`testUnresolvedHostConnection`) might need to increase their timeout up to ~20 sec.

And please run tests to verify changes.

File: transport-parent/transport-netty/pom.xml (hunk 1 above)
Comment (Sm0keySa1m0n) [reply transport-parent/transport-netty/pom.xml:24]: Ahh, I was looking for what defined that version but couldn't find it. I'll update that, thanks.

**Comment by Sm0keySa1m0n** [comment]
I keep getting `java.io.IOException: An established connection was aborted by the software in your host machine`, do you know why that might be happening?

**Comment by segabriel** [comment]
https://github.com/scalecube/scalecube-cluster/pull/354

**Pull request closed without merging**